charts/portworx-essentials/v1.0.0/README.md

```diff
@@ -0,0 +1,39 @@
+# Portworx
+
+## **Pre-requisites**
+
+Use this Helm chart to deploy [Portworx](https://portworx.com/) and [Stork](https://docs.portworx.com/scheduler/kubernetes/stork.html) to your Kubernetes cluster.
+
+Prerequisites
+
+Refer to the [Install Portworx on Kubernetes via Helm](https://docs.portworx.com/portworx-install-with-kubernetes/install-px-helm/#pre-requisites) page for the list of prerequisites.
```

Comment:
let's remove this as these are no longer relevant. This chart will get installed from Rancher UI itself. So don't need to talk about helm.